frontend/src/static/js/pages/styles/MediaPage.scss

```
@@ -557,15 +557,15 @@ $widerLayoutVersion: true; // @todo: Choose dynamically the layout version, base
 
 .viewer-container .player-container.viewer-pdf-container,
 .viewer-container .player-container.viewer-attachment-container {
-	background-color: #f5f5f5;
+	background-color: var(--bg-surface-muted);
 	min-height: 0.5 * 259px;
 	// padding-top: 20.235%;
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)